packages/blocks/src/blocks/CoreImage.tsx

```diff
@@ -0,0 +1,87 @@
+import { gql } from '@apollo/client';
+import React from 'react';
+import { useBlocksTheme } from '../components/WordPressBlocksProvider.js';
+import {
+  BlockWithAttributes,
+  ContentBlock,
+} from '../components/WordPressBlocksViewer.js';
+import { getStyles } from '../utils/index.js';
+
+export type CoreImageFragmentProps = ContentBlock & {
+  attributes: {
+    align?: string;
+    alt?: string;
+    anchor?: string;
+    borderColor?: string;
+    caption?: string;
+    className?: string;
+    width?: string;
+    url?: string;
+    title?: string;
+    style?: string;
+    src?: string;
+    sizeSlug?: string;
+    rel?: string;
+    lock?: string;
+    linkTarget?: string;
+    linkDestination?: string;
+    linkClass?: string;
+    id?: string;
+    href?: string;
+    height?: string;
+    cssClassName?: string;
+  };
+};
+
+export function CoreImage(props: BlockWithAttributes<CoreImageFragmentProps>) {
```

Comment:
`BlockWithAttributes` is not needed.
use `CoreImageFragmentProps` instead

Reply:
Thank you!